src/main/java/seedu/address/logic/commands/ShowCommand.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.IdentityCardNumberMatchesPredicate;
+import seedu.address.model.person.Person;
+
+
+
+/**
+ * Shows the user a note of a person in the address book.
+ * If no IC is given, the displayed note panel will be cleared.
+ */
+public class ShowCommand extends Command {
+
+    public static final String COMMAND_WORD = "show";
+    public static final String MESSAGE_SHOW_NOTE_SUCCESS = "Displayed note of person: %1$s";
+
+    public static final String MESSAGE_CLEAR_NOTE_SUCCESS = "Note cleared.";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows the note of the person whose IC matches the "
+            + "specified IC (case-insensitive) and displays it.\n"
```

Comment:
do we not need to say valid ic number? cause I saw the change in every file

Reply:
probably not too important, but I can add it in if you feel it's necessary